src/vs/workbench/contrib/chat/browser/chatEditorController.ts

```diff
@@ -491,7 +491,7 @@ class DiffHunkWidget implements IOverlayWidget {
 
 
 	constructor(
-		readonly entry: IModifiedFileEntry,
+		readonly entry: IModifiedTextFileEntry,
```

Comment:
This class only deals with editors for text files, editors for notebooks are already handled separately.